generated/Destinations.js

```diff
@@ -155,6 +155,8 @@ export const TIKTOK_ADS_NAME = 'TIKTOK_ADS';
 export const TIKTOK_ADS_DISPLAY_NAME = 'TikTok Ads';
 export const TVSQUARED_NAME = 'TVSQUARED';
 export const TVSQUARED_DISPLAY_NAME = 'TVSquared';
+export const USERPILOT_NAME = 'USERPILOT';
```

Comment:
@koladilip, I believe the files under `generated` directory, should not be modified, correct?